### DISCUSSION TITLE
Distiguishing between databases in graphql api

### DISCUSSION THREAD

**Discussion by nizalsha** [hasura/graphql-engine#9835] [Question]
Hi,

I have multiple db's connected in my hasura cloud, everything in accessed through the graphqlapi and key, and the the query root is common. This makes error messages when adding databases with same schema, table names. So my question is how can i distinguish the databases separatley in my graphql api? Is there any way to access the specific db and query from that db only ?

**Comment by tirumaraiselvan** [comment]
You can "namespace" the different databases by adding some graphql customizations. This way all tables for that database will appear under the namespace field.  This can be done by editing the database:

<img width="807" alt="Screenshot 2023-08-22 at 7 13 48 PM" src="https://github.com/hasura/graphql-engine/assets/8663570/703cb31f-22bb-4c3f-81e7-dcd63bdae807">
